src/nyc_trees/apps/core/urls.py

```diff
@@ -4,16 +4,11 @@
 from __future__ import division
 
 from django.conf.urls import patterns, url
-from django.views.generic import TemplateView
 
-from django_tinsel.decorators import route
+from apps.core.routes import js_settings
 
 
-# These URLs have the prefix 'quiz/'
 urlpatterns = patterns(
     '',
-    url(r'^js-settings/$',
-        route(GET=TemplateView.as_view(template_name='core/settings.js',
-                                       content_type='application/javascript')),
-        name='js_settings'),
+    url(r'^js-settings/$', js_settings, name='js_settings'),
```

Comment:
I see. I thought we had decided to save the round trip by inlining it, but that's probably not a big issue since it will load in parallel with the JS bundle.

When we make a partial to load the page-specific JavaScript we should move all JavaScript loading there so we don't load JavaScript unnecessarily on JavaScript-less pages.